include/swift/SIL/SILConstants.h

```diff
@@ -239,6 +248,11 @@ class SymbolicValue {
 
   APFloat getFloatValue() const;
 
+  static SymbolicValue getString(const StringRef &string,
```

Comment:
Please pass the StringRef by value, it is a small POD type.  Please add a comment that the bytes are in UTF8 to both methods.

Reply:
Thanks, good catch! Done in https://github.com/apple/swift/pull/17235/commits/013f7a48819b2daee4a4f09e953fb7e33ccb96cd.